src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        Ball[] balls = new Ball[3];
+        for (int i = 0; i < balls.length; i++) {
+            balls[i] = new Ball();
+            System.out.println(lottery.getRandomBall(balls[i]));
```

Comment:
two points:

1. `3` is a magic number, you need to create a dedicated constant for the number of balls
2. you don't need to create an array for storing the balls, since your task is just to print them

```suggestion
        private static final int NUMBER_OF_BALLS = 3;
        // method declaration goes here
        for (int i = 0; i < NUMBER_OF_BALLS; i++) {
            System.out.println(lottery.getRandomBall());
```

this change should be done after you address my [comment](https://github.com/mate-academy/jv-lottery/pull/1088/files#r1290240047)